python-sdk-sample/src/main.py

```diff
@@ -8,4 +8,122 @@
 Copyright Contributors to the Zowe Project.
 """
 
-print("Hello world!")
+from zowe.core_for_zowe_sdk import SdkApi
+
+
+class SampleSdk(SdkApi):
+    """
+    Class used to represent the sample API.
+
+    ...
+
+    Attributes
+    ----------
+    connection
+        Connection object
+    """
+
+    def __init__(self):
+        """
+        Construct a sample SDK object.
+
+        Parameters
+        ----------
+        connection
+            The connection object
+        """
+        super().__init__({"host": "jsonplaceholder.typicode.com", "user": "fake", "password": "fake"}, "/", logger_name=__name__)
```

Comment:
It seems that the sample SDK depends on features that have not been published to the Pypi registry.
Should we get a 1.0.0-dev16 out there before we merge this PR?